### PR TITLE
Adding user-docs directory

### PR DESCRIPTION
**Pull request by XMRZombie**
Documentation inspired from bisq for haveno

### CODE REVIEW

**Comment by woodser** [comment]
Thanks, but actually this would be perfect to add to the https://github.com/haveno-dex/haveno-docs repository, so it's presented through our official docs: https://docs.haveno.exchange/.

Maybe just add a new section there for User Docs which all of this can go under for now?

This would be a great addition to those docs!

File: docs/user-docs/get_started.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- In order to trade monero on Haveno, you've got to set up some way to send or receive other funds. Haveno only handles the monero side of a trade—the other side is handled through fiat payment services (banks, money orders, cash) or altcoin wallets.
+
+Not sure which kind of payment account to set up? There's a full list of [payment methods](https://github.com/haveno-dex/Haveno/blob/master/docs/payment_methods.md) on the wiki.
+
+
+### 4. Do a Trade
+
+Making an offer will usually get you a better price and more control (e.g., setting payment method and deposit percentage), but taking an offer can be more convenient.
+To see both sides of a Haveno trade at the same time, side-by-side: be sure to check out our [trading recommendations](trade-recommendations/payment_methods/README.md) and [linked tutorials made by the community](https://haveno-reto.com/#posts).
```
Comment (woodser) [inline docs/user-docs/get_started.md:29]: This link needs to be removed, but could be replaced by links directly to the tutorials.

**Comment by boldsuck** [comment]
Now I know why I couldn't find your PR ;-) I searched [haveno-docs](https://github.com/haveno-dex/haveno-docs)
You have lost [Moneta](https://github.com/boldsuck/haveno-reto-docs/blob/main/docs/trade-recommendations/payment_methods/Moneta.md)

I'm in the process of creating the side menu for mkdocs. Workflow complained about deploy.yml. I updated it.

**Comment by XMRZombie** [comment]
I close this PR to modify links and reuploading that on haveno-docs

**Pull request closed without merging**